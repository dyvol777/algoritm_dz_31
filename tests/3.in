2 2
3 4 1 a
1 0 3 bwsess