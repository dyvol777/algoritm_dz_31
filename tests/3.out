something going wrong! - incorrect value!
